2020-APR/answer.md

```diff
@@ -0,0 +1,203 @@
+<!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=6 orderedList=false} -->
+
+# AACS1084 April 2020 Answers
+
+[Link to the paper](https://eprints.tarc.edu.my/14889/1/AACS1084.pdf)
+
+- [Question 1](#question-1)
+- [Question 2](#question-2)
+- [Question 3](#question-3)
+- [Question 4](#question-4)
+
+## Answers
+
+### Question 1
+
+a) (i)
+
+Structure is a data structure that allows user to store a group of variables with different data types.
+
+b)
+
+To group several related variables taht is different in data types into one place.
```

Comment:
Structure are commonly used to to define records to be stored in a file, where each element in a structure represents a field within a record.